pkg/agent/util/net_windows.go

```diff
@@ -49,6 +50,7 @@ const (
 	MetricHigh    = 50
 
 	AntreaNatName = "antrea-nat"
+	LocalVMSwitch = "antrea-switch"
```

Comment:
If it is accessed only in this file, use a private variable. So does "OVSExtensionName".

Reply:
OVSExtensionName - This can be local variable. 
LocalVMSwitch - is referenced in agent_windows.go, so it will be public.